### PR TITLE
[Tooling] Makefile checkers when starting localnet

### PR DESCRIPTION
**Pull request by Olshansk**
## Type of change

Select one or more:

- [x] New feature, functionality or library
- [ ] Bug fix
- [ ] Code health or cleanup
- [ ] Documentation
- [ ] Other (specify)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -240,7 +266,7 @@ docker_wipe: check_docker warn_destructive prompt_user ## [WARNING] Remove all t
 ########################
 
 .PHONY: localnet_up
-localnet_up: proto_regen localnet_regenesis ## Starts localnet
+localnet_up: check_docker_ps check_kind_context proto_regen localnet_regenesis ## Starts localnet
```
Comment (red-0ne) [inline Makefile:269]: :+1:

File: Makefile [hunk 2]
```diff
@@ -129,6 +129,32 @@ check_docker:
...
 	}
+.PHONY: check_kind
+# Internal helper target - check if kind is installed
+check_kind:
+	@if ! command -v kind >/dev/null 2>&1; then \
+		echo "kind is not installed. Make sure you review build/localnet/README.md and docs/development/README.md  before continuing"; \
+		exit 1; \
+	fi
+
+.PHONY: check_k
```
Comment (red-0ne) [inline Makefile:140]:
```suggestion
.PHONY: check_docker_ps
```
?

**Review by red-0ne — APPROVED** [review]
NIT changes and preemptively approving :+1:

**Review by okdas — APPROVED** [review]
Lgtm. We didn't have that to allow other k8s distributions, but I'm happy if we force devs to use `kind` only :)

**Pull request merged**